src/azure-cli/azure/cli/command_modules/appservice/_help.py

```diff
@@ -353,7 +353,18 @@
 short-summary: Create a Managed Certificate for a hostname in a function app.
 examples:
   - name: Create a Managed Certificate for $fqdn.
-    text: az functionapp config ssl create --resource-group MyResourceGroup --name MyWebapp --hostname $fqdn
+    text: az functionapp config ssl create --resource-group MyResourceGroup --name MyFunctionApp --hostname $fqdn
+"""
+
+helps['functionapp config ssl sync'] = """
+type: command
+short-summary: Sync an imported certificate on a functionapp with its object in KV.
```

Comment:
try not to use acronyms like KV - use full name "KeyVault"

Reply:
this is not syncing cert - this is syncing the webapp to the latest version of the certificate from KeyVault; that is associated with the webapp/function name. Please update the summary to reflect this.